projects/packages/publicize/src/class-publicize.php

```diff
@@ -253,12 +253,13 @@ public function get_connections( $service_name, $_blog_id = false, $_user_id = f
 	/**
 	 * Get all connections for a specific user.
 	 *
-	 * @param arayy $args Arguments to run operations such as force refresh and connection test results.
+	 * @param array $args Arguments to run operations such as force refresh and connection test results.
 
 	 * @return array
 	 */
 	public function get_all_connections_for_user( $args = array() ) {
-		if ( isset( $args['clear_cache'] ) && $args['clear_cache'] ) {
+		if ( ( isset( $args['clear_cache'] ) && $args['clear_cache'] )
+		|| ( isset( $args['run_test_results'] ) && $args['run_test_results'] ) ) {
```

Comment:
Did you want to change this to `test_connections` @manzoorwanijk? I think it's maybe a better name.